src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
Yes, I assumed it would be changed in both places.

I am trying to come up with a name to replace `length`.

We have agreed on `count`, but I am not particularly happy with that choice for the reasons I mentioned.